src/BridgedRoom.ts

```diff
@@ -611,11 +611,13 @@ export class BridgedRoom {
             };
             return ghost.sendMessage(this.MatrixRoomId, matrixContent, channelId, eventTS);
         } else if (message.files) { // A message without a subtype can contain files.
+            const uploadSize = this.main.config.homeserver.max_upload_size;
```

Comment:
Can we make it `maxUploadSize`? Makes the check line clearer (`file.size > uploadSize`)